app/components/candidate_interface/degree_required_component.rb

```diff
@@ -0,0 +1,73 @@
+class CandidateInterface::DegreeRequiredComponent < ViewComponent::Base
+  CANDIDATE_GRADES = {
```

Comment:
Think this is missing 'First class honours'?

Reply:
Strange... it was definitely in there at some point, can't think when it was stripped out. Will add back in, good spot.